source/catalog.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Фотографии пользователей</title>
```

Comment:
Единообразия ради можно оформить тайтл по образцу титульной:
```suggestion
    <title>Pink - Фотографии пользователей</title>
```
Аналогично в форме